packages/compile/src/generate/code-gen.js

```diff
@@ -157,19 +150,19 @@ ${chunkedFunctions('evalLevels', Model.levelVars(), '  // Evaluate levels.')}
   // Input/output section
   //
   function emitIOCode() {
-    let headerVars = outputAllVars ? expandedVarNames(true) : spec.outputVars
-    let outputVars = outputAllVars ? expandedVarNames() : spec.outputVars
+    let headerVarNames = outputAllVars ? expandedVarNames(true) : spec.outputVarNames
+    let outputVarIds = outputAllVars ? expandedVarNames() : spec.outputVars
```

Comment:
I renamed these two variables to emphasize the difference, but otherwise it's the same as you had it.